bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/util/ImprovedSkyCalcMethods.java

```diff
@@ -922,9 +922,9 @@ protected static double lst(double jd, double longit) {
             sid_g = sid_g + 1.0027379093 * ut - longit / 24.;
             sid_int = (long) sid_g;
             sid_g = (sid_g - sid_int) * 24.;
-    //        if (sid_g < 0.) {
-    //            sid_g = sid_g + 24.;
-    //        }
+           if (sid_g < 0.) {
```

Comment:
I wonder if this affects `lucuma-core`, we may have ported some of the code there

Reply:
It's worth a look.